json/README.md

```diff
@@ -50,13 +50,13 @@ $ ../gradlew run
 ```
 
 It first creates a GemFire client with a proxy client cache. 
-Then it converts a number of JSON strings to binary form and put to the `example-region`.
+Then it converts a number of JSON strings to a binary form called JsonDocument and put them to the `example-region`.
```

Comment:
change to: Then it converts a number of JSON strings to JsonDocuments and puts them into the...